src/components/announcements-listing/AnnouncementsList.tsx

```diff
@@ -0,0 +1,169 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
🔧 Naming convention should be `/announcementsList/index.tsx`

Reply:
I think I changed it in my branch (merged now), so there will probably be conflicts